R/db.R

```diff
@@ -309,6 +314,9 @@ write_data_safely <- function(dbconn, tblname, df, key, safe_columns) {
     to_chars <- unname(vapply(df, function(x) is.factor(x) || is.ordered(x) || is.logical(x), logical(1)))
     df[to_chars] <- lapply(df[to_chars], as.character)
 
+    ## Add timestamp
+    df$last_cached_at <- Sys.time()
```

Comment:
Won't this be messed up if people use different time zones for their local machine?